test/e2e/upgrade/cidisruptiontester/cidisruptiontester.go

```diff
@@ -0,0 +1,77 @@
+package cidisruptiontester
+
+import (
+	"time"
+
+	"github.com/openshift/origin/pkg/monitor/backenddisruption"
+	"github.com/openshift/origin/test/extended/util/disruption"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/framework/service"
+	"k8s.io/kubernetes/test/e2e/upgrades"
+)
+
+// ciDisruptionUpgradeTest tests the actual CI cluster where tests are running is able to reach an external service. This
+// is used to compare if we're actually observing disruption in the cluster under test, or if the CI cluster itself
+// is losing networking.
+//
+// The service in question is maintained by the TRT team, and running on the DPCR cluster in
+// the trt-monitoring namespace. (ci-disruption-tester svc)
+type ciDisruptionUpgradeTest struct {
+	// filled in by pre-setup
+	jig                 *service.TestJig
+	tcpService          *v1.Service
+	unsupportedPlatform bool
+	hostGetter          *backenddisruption.SimpleHostGetter
+
+	backendDisruptionTest disruption.BackendDisruptionUpgradeTest
+}
+
+func NewCIDisruptionWithNewConnectionsTest() upgrades.Test {
+	ciDisruptTest := &ciDisruptionUpgradeTest{
+		hostGetter: backenddisruption.NewSimpleHostGetter(""), // late binding host
+	}
+	backend := backenddisruption.NewSimpleBackend(
+		"https://trt-ci-disruption-tester.dptools.openshift.org",
+		"ci-cluster-network-liveness",
+		"/",
+		backenddisruption.NewConnectionType)
+	allowed := 1 * time.Second
+	ciDisruptTest.backendDisruptionTest =
+		disruption.NewBackendDisruptionTestWithFixedAllowedDisruption(
+			"[sig-trt] CI cluster remains able to communicate with an external service using new connections",
+			backend,
+			&allowed, // We'll let one second slide without reporting a problem
+			"CI cluster where tests are running may have network issues (not the cluster under test)",
```

Comment:
This message is identical to the one above it for new Connections, I suggest making it different so we can tell which test caused the message.

Reply:
My suggestion: make these flake for now (I agree with out that no one but TRT will know to look at these in case there are disruption failures).  But we can keep tabs on the test (with no risk of it failing payloads) when there are disruption failures to see if this test also fails too -- on our investigation, we can make the determination if the test should or should not have failed and make adjustments as necessary.

After some time, when we think the test is really reliable, we can unflake it.

Reply:
It's merged as a fail but I will keep an eye on it. If it looks like it's firing when it shouldn't, I will adjust to a flake somehow. I'd really like it to be in people's faces when they look at a PR with disruption, but if I see any reasonable number of it failing without other disruption, I'm with you, it needs to be a flake and something we interpret behind the scenes.